packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -678,6 +683,174 @@ function wait(time) {
   });
 }
 
+function handleErrorMsg(err) {
+  if (err?.errorMessage) {
+    cliux.print(`Error: ${err.errorMessage}`, { color: 'red' });
+  } else if (err?.message) {
+    cliux.print(`Error: ${err.message}`, { color: 'red' });
+  } else {
+    console.log(err);
+    cliux.print(`Error: ${messageHandler.parse('CLI_EXPORT_CSV_API_FAILED')}`, { color: 'red' });
+  }
+  process.exit(1);
+}
+async function apiRequestHandler(org, queryParam = {}) {
+  const headers = {
+    authtoken: configHandler.get('authtoken'),
+    organization_uid: org.uid,
+    'Content-Type': 'application/json',
+    api_version: 1.1,
+  };
+
+  return await new HttpClient()
+    .headers(headers)
+    .queryParams(queryParam)
+    .get(`${configHandler.get('region')?.cma}/organizations/${org.uid}/teams`)
+    .then((res) => {
+      const { status, data } = res;
+      if(status===200) {
+        return data;
+      } else {
+        cliux.print(`${data.error_message || data.message || data.errorMessage}`, { color: 'red' });
+        process.exit(1);
+      }
+    })
+    .catch((error) => {
+      handleErrorMsg(error);
+    });
+}
+
+async function exportOrgTeams(managementAPIClient, org) {
+  let teamsObjectArray = [];
+  let skip = 0;
+  let limit = config.limit || 100;
+  do {
+    const data = await apiRequestHandler(org, { skip: skip, limit: limit, includeUserDetails: true });
+    skip += limit;
+    teamsObjectArray = [...teamsObjectArray, ...data?.teams];
+    if (skip > data?.count) break;
```

Comment:
@cs-raj, Can you validate this case with 100, 200 data count ? I think we can avoid one more iteration using skip >= data?.count.

Reply:
Validated @aman19K by changing the limits